### PR TITLE
updates of markup, adding questions 23 and 27, 28

### PR DESCRIPTION
**Pull request by StenGruener**
@aorzelskiGH @BirgitBoss  I have added two more questions, also hiddnen ID's for the question for an easy tracibility.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,10 +74,32 @@ e.g. the indication of a lifecycle state of a value by “as-specified” or “
...
 (Answered: 2020-08-13)
 
+**What are best practices for creating custom IRI identifiers for generic concepts.** <a name="id18"></a><!-- ID: 18 -->
 
+What are best practices for creating custom IRI identifiers for generic concepts.
+We advise using “https://admin-shell.io/” prefix for those identifiers (see guides via https://github.com/admin-shell-io/id/) for generic concepts. Note that we advise to use “https:” protocol and avoid adding “www.” subdomain into admin-shell.io naming scheme.
+Furthermore, in practical implementations we advise to “filter” out protocol and schema for comparing IRI identifiers. For example, following IDs should be considered equal:
+-	https://admin-shell.io/some_id_example
+-	http://admin-shell.io/some_id_example
+-	ftp://admin-shell.io/some_id_example
```
Comment (BirgitBoss) [inline README.md:89]: ftp really equal to https etc.?

File: README.md [hunk 2]
```diff
@@ -74,10 +74,32 @@ e.g. the indication of a lifecycle state of a value by “as-specified” or “
...
+-	http://www.admin-shell.io/some_id_example
+(Answered: 2020-08-19)
+
+**Which submodel shall contain the serial number of a device?** <!-- ID: 23 -->
+
+Serial number is a submodel element of “Identification” submodel referenced in AAS for an asset with Type=Instance, i.e. instance’s AAS. 
+(Answered: 2020-08-19)
+
+**How to refer to semantic concepts of existing standards like VDI 2770 properties or OPC UA companion specifications (e.g. “Serial number” property from OPC UA DI companion spec)?** <!-- ID: 27, 28 -->
+Map concepts to “admin-shell.io” namespace (see [custom identifier best practices](#id18)). Within the concept description itself use “sourceOfDefinition” or “isCaseOf” to link to “original” concept.
```
Comment (BirgitBoss) [inline README.md:100]: The link custom identifier best practices is very strange... does not jump to the question but to complete FAQ list.

File: README.md (hunk 1 above)
Comment (StenGruener) [reply README.md:89]: Idea was to ignore the protocol-specific schema to make implementation practical

Comment (BirgitBoss) [reply README.md:89]: 2020-08-24 Meeting Exemplarische Modellierungen: no change needed, this is what should be done when comparing ids.

File: README.md [hunk 3]
```diff
@@ -74,10 +74,32 @@ e.g. the indication of a lifecycle state of a value by “as-specified” or “
 e.g. the indication of multiplicity of allowed components for modelling by “greater than 0” and others.
 (Answered: 2020-08-13)
 
-**Where are examples of AAS available?**
+**Where are examples of AAS available?** <!-- ID: 16 -->
 
 Examples of several suppliers are found at: http://admin-shell-io.com/samples/ 
 (Answered: 2020-08-13)
 
+**What are best practices for creating custom IRI identifiers for generic concepts.** <a name="id18"></a><!-- ID: 18 -->
```
Comment (BirgitBoss) [inline README.md:82]: 2020-08-24 Meeting Exemplarische Modellierung: check whether <a name=...> still valid, seems to be obsolete.

**Pull request merged**